### ISSUE TITLE
How do you test the accuracy of your model?

### ISSUE THREAD

**Issue by Patricia1019** [pengzhiliang/MAE-pytorch#47]
Hello, I can find run_mae_vis.py ,but I can't find the code to test the accuracy, can you provide one? Thanks~

**Comment by pengzhiliang** [comment]
Ok, maybe you can directly run the following command:
```bash
python run_class_finetuning.py \
    --model vit_base_patch16_224 \
    --data_path ${DATA_PATH} \
    --finetune ${MODEL_PATH} \
    --output_dir ${OUTPUT_DIR} \
    --batch_size 128 \
    --opt adamw \
    --opt_betas 0.9 0.999 \
    --weight_decay 0.05 \
    --epochs 100 \
    --eval
```
params:
--finetune: the model from fine-tuning
--eval : [check here](https://github.com/pengzhiliang/MAE-pytorch/blob/8b3ab30f99000ac56b2469de6fd48341ed799a21/run_class_finetuning.py#L447)
